docs/TuxxAnimations.md

```diff
@@ -111,93 +118,138 @@ A numeric id can be specified, in which case all elements will receive that key.
+The vector property allows you set where your animation will orignate from and the magnitude of that entrance and exit to that origin. The vector property ccepts an object and expects two keys within that object: `origin` and `magnitude`. Magnitude accepts a number value and Origin accepts any single item of or any combination of two items from the the following list of acceptable origins:  'top', 'bottom', 'left' and 'right'. For example, if you wanted your elements to originate from the bottom right corner of the containing animation component with a magnitude of 50 pixels, it's as easy as:
+
+```javascript
+  <Fade duration={3000} delay={1000} easing="easeInOutBack" vector={{origin: 'bottom right', magnitude: 50}}>
+    <h1>Hello World</h1>
+  </Fade>
+```
+
+
 ##### <a id="animationProps-custom"></a>Custom Property - type: OBJECT - optional [#](#animationProps-custom)
-The custom property accepts an object as an input with any CSS properties you wish the animation to include in addition to the default CSS transitions inherent to the particular component. For Example, if you wanted the element you've wrapped with a Fly animation component to also animate to the color red and increase its font size to 14px, it's as easy as:
+The custom property accepts an object as an input with any CSS properties you wish the elements within the animation component to transition to on enter. For example, if you wanted the element you've wrapped with a Fade animation component to also animate to the color red and increase its font size to 50px, it's as easy as:
 
 ```javascript
-  <Fly duration={3000} delay={1000} easing="easeOutInBack" custom={{'color': 'red', 'font-size': '14px'}}>
+  <Fade duration={3000}
+        delay={1000}
+        easing="easeInOutBack"
+        vector={{origin: 'bottom right', magnitude: 40}} custom={{'color': 'red'}}>
     <h1>Hello World</h1>
-  </Fly>
+  </Fade>
 ```
 
-Finally, let's see how the default Fly animation looks now that it has been customized via its 4 default properties:
+**NOTE:** The CSS set in the custom property will overide the normal CSS of the elements nested within the animation componenet. Additionally, the CSS set in the custom property will transition back to the inhereted CSS on leave.
+
+Let's see how the default Fade animation looks now that it has been customized via these 5 simple default properties:
+
+![Animation 2](http://cdn.makeagif.com/media/2-07-2015/JfkPsC.gif)
 
-![Animation 2](http://cdn.makeagif.com/media/1-12-2015/9Hqmtk.gif)
+##### <a id="animationProps-enter"></a>Enter Property - type: OBJECT - optional [#](#animationProps-enter)
+The enter property allows you to completely custom how your animation will look upon entering the DOM. The enter property accepts an object as an input that can contain any of the 5 simple properties described above as keys with their corresponding accepted inputs as values.
```

Comment:
What are the "... accepted inputs as values."? Are they already listed out somewhere? Should they be listed out?